File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_1/The solution/CPP/Q_2.cpp
#include <iostream>
using namespace std;

int main()
{
    int num1, num2;

    // Read inputs from user
    cout << "Enter first number: ";
    cin >> num1;
    cout << "Enter second number: ";
    cin >> num2;

    // Calculate the difference
    int diff = num1 - num2;
    if(diff >= 0)
        cout << "The difference is: " << diff << endl;
    else
        cout << "The difference is: " << diff * -1 << endl; // بعكس اشارة الفرقبضربة ف سالب واحد
    return 0;
}

File: Level_1/Term_2/Progrmming 2/Sheets/Sheet_1/Answers/src/com/company/Q_2.java
package com.company;

import java.util.Scanner;

public class Q_2 {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        int num1=0, num2=0, num3=0;

        // Read 3 numbers from user
        for(short i = 1; i<=3; i++)
        {
            System.out.printf("Enter number[%d]: ", i);
            switch(i){
                case 1 -> num1 = input.nextInt();
                case 2 -> num2 = input.nextInt();
                case 3 -> num3 = input.nextInt();
            }
        }
        //Print result
        System.out.printf("(%d * %d * %d)^2 = %d", num1, num2, num3, (int)Math.pow(num1*num2*num3,2));
    }
}
